src/main/java/com/amazon/jenkins/ec2fleet/CloudConstants.java

```diff
@@ -0,0 +1,33 @@
+package com.amazon.jenkins.ec2fleet;
+
+class CloudConstants {
+
+    static final String EC2_INSTANCE_TAG_NAMESPACE = "ec2-fleet-plugin";
+    static final String EC2_INSTANCE_CLOUD_NAME_TAG = EC2_INSTANCE_TAG_NAMESPACE + ":cloud-name";
+    static final boolean DEFAULT_PRIVATE_IP_USED = false;
+
+    static final boolean DEFAULT_ALWAYS_RECONNECT = false;
+
+    static final int DEFAULT_IDLE_MINUTES = 0;
+
+    static final int DEFAULT_MIN_SIZE = 1;
```

Comment:
This will always leave 1 instance running unless maxTotalUses is set and met i.e. customer might pay for an idle instance due to this default. Was there a reason to change the default from 0 to 1?

Reply:
Per our offline sync, I understand jelly has a defualt of 1. Can we just make the default 0 ? 🤔

Reply:
Yup, I'll also change the Jelly to reflect this default. I think it makes more sense to not have idle instances.